src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -80,7 +80,6 @@ public void SendAsync_SimpleGet_Success()
 
         [OuterLoop]
         [Fact]
-        [OuterLoop]
```

Comment:
Why is this being removed? Tests targeting 3rd party servers should remain in outerloop.

Reply:
There was 2 `OuterLoop` attribute I just deleted one, so that should be fine